Gems/Atom/Tools/MaterialEditor/Code/Source/MaterialEditorApplication.cpp

```diff
@@ -511,11 +346,6 @@ namespace MaterialEditor
         AzFramework::Application::Stop();
```

Comment:
Update all of your base class calls in both applications to AtomToolsAppication:: or Base::